console.go

```diff
@@ -36,6 +37,13 @@ type Console struct {
 	slavePath string
 }
 
+// IsTerminal returns true if fd is a terminal, else false
+func IsTerminal(fd uintptr) bool {
+	var termios syscall.Termios
+	_, _, err := syscall.Syscall6(syscall.SYS_IOCTL, fd, syscall.TCGETS, uintptr(unsafe.Pointer(&termios)), 0, 0, 0)
```

Comment:
Given the fact that you don't use 4th, 5th and 6th parameters, you should use `syscall.Syscall()` instead of `syscall.Syscall6()`